source/clear-linux/guides/deploy-at-scale.rst

```diff
@@ -0,0 +1,342 @@
+.. _deploy-at-scale.rst:
+
+Deploying |CL| at Scale
+#######################
+
+Once you are comfrotable with `Clear Linux concepts`_, your next step
+as a system administraotr may be needing to understand how to deploy 
+|CL| at scale in your enoviornment.  
+
+Throughout this document the term *endpoint* will be used to generally refer 
+to a system targeted for |CL| installation, whether that is a datacenter 
+system or unit deployed in field.  
+
+.. note::
+
+    This is not a replacement or blueprint for designing your IT operating 
+    environment. These are simply recommendations that should be  implemented 
+    with consideration. 
+    
+    Your |CL| deployment should complement the 
```

Comment:
Not sure this needs to be included

Reply:
I feel it is an important in order to set the expectations for the readers, who may be sysadmins, and looking for a step-by-step whitepaper on how to deploy a generic IT environment which can scale. 

Especially important for non-sysadmins who may otherwise read these ideas and start implementing in conflict with their sysadmin team(s).